eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Ratify.hs

```diff
@@ -135,6 +140,51 @@ accepted RatifyEnv {reStakePoolDistr = PoolDistr poolDistr} gas =
       HardForkInitiation {} -> 101 % 100
       _ -> spoThreshold
 
+dRepAccepted :: forall era. RatifyEnv era -> GovActionState era -> Bool
+dRepAccepted RatifyEnv {reDRepDistr} GovActionState {gasDRepVotes, gasAction} =
+  case dRepAcceptedRatio reDRepDistr gasDRepVotes gasAction of
+    Nothing -> True -- TODO: change this when we are ready to consider dReps on sanchonet
+    Just ratio -> ratio > dRepThreshold
```

Comment:
According to the spec, it should also return True if the threshold is set to zero:
https://github.com/input-output-hk/formal-ledger-specifications/blob/master/src/Ledger/Ratify.lagda#L225

Reply:
and it looks like this is supposed to be a non-strict inequality
```suggestion
    Just ratio -> ratio >= dRepThreshold
```

Reply:
Good catch, I missed that. Thanks